core/src/test/java/com/scalar/db/storage/cassandra/CassandraAdminTestBase.java

```diff
@@ -43,20 +52,42 @@
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
-public class CassandraAdminTest {
+/**
+ * Abstraction that defines unit tests for the {@link CassandraAdmin}. The class purpose is to be
+ * able to run the {@link CassandraAdmin} unit tests with different values for the {@link
+ * CassandraAdmin}, notably {@link CassandraAdmin#METADATA_KEYSPACE}.
+ */
+public abstract class CassandraAdminTestBase {
```

Comment:
The `CassandraAdminTest` was refactored the same way as the other storage Admin classes. cf https://github.com/scalar-labs/scalardb/pull/670 https://github.com/scalar-labs/scalardb/pull/671